.github/workflows/prbot.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           app-id: ${{ secrets.GH_APP_ID }}
           private-key: ${{ secrets.GH_APP_PRIVATE_KEY }}
           owner: coveo
-          repo: ui-kit-prs
```

Comment:
https://github.com/actions/create-github-app-token/tree/31c86eb3b33c9b601a1f60f98dcbfd1d70f379b4/?tab=readme-ov-file#create-a-token-for-multiple-repositories-in-the-current-owners-installation

Reply:
I wonder why they didn't make this an array...

Reply:
Arrays is special in yml let's say, and hard to interpret.
Often you'll see toJson and the like in GHA to go thru the hoop and cast that to string or other primitive (string, number, boolean)